README.md

```diff
@@ -158,6 +158,15 @@ called after each deployment to each environment.
 
 The smoke tests are to be non-destructive (i.e. have no data impact, such as not creating accounts) and a subset of component level functional tests.
 
+#### Docker Test Build
+
+A Docker Test Build can be enabled providing static unit tests, security checks & Sonar scanning for builds from Master if needed. This stage is always run on PR so in most cases does not need enabling for Master builds if following common git workflow. 
```

Comment:
I think from https://github.com/hmcts/cnp-plum-recipes-service/commit/0688c0af15360c0707c6ca8011afc4cfe2c40b98 these are still run. It only provides a test docker image if enabled